tests/issues.targets

```diff
@@ -194,10 +194,7 @@
             <Issue>times out</Issue>
         </ExcludeList>
         <ExcludeList Include="$(XunitTestBinBase)/JIT/opt/Tailcall/TailcallVerifyWithPrefix/*">
-            <Issue>Uses illagal il tailcall pop ret</Issue>
-        </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)/JIT/Methodical/tailcall_v4/hijacking/*">
-            <Issue>13828</Issue>
+            <Issue>Uses illegal il tailcall pop ret</Issue>
```

Comment:
I see all the invalid sequences in the test commented out with:

```
//            tail.      // tail.call, pop, ret sequence is never valid for .NET Core (but is accepted by .NET x64)
```

It would nice to delete the invalid tests so that we do not have to be excluding them forever.

Reply:
I think I would prefer to do that in a separate PR, where we remove all tests of a certain type. That way if we bring the feature back it is easier to undo. We have had issues with this in the past, ie native varargs.

Reply:
I can put it in a separate commit and then the PR can be rebased so I don't have to open another PR.

Reply:
It seems like these tests are actually already not being run, so I will just clean it up as well as I can.

Reply:
However there are other tests that are incompatible with many platforms. For instance, tests that implicit tailcalls are performed and a test that pinvokes kernel32. It does not seem we will be able to reenable this test any time soon. I guess I can remove those or try to condition them behind checks. What do you suggest?

Reply:
@jkotas the test tries a pinvoke to kernel32!Sleep. Is there a pinvokes declaration for some coreclr function I could use that would "just work" on all our supported platforms?

Reply:
Can the test call `Thread.Sleep`?

Reply:
No, it is specifically testing `tail.` prefixed pinvoke call.

Reply:
Unless `Thread.Sleep` is just a public pinvoke?

Reply:
For Unix, you can use `printf("Hello")` in `[DllImport("libc")]`